apps/login/src/ui/SessionItem.tsx

```diff
@@ -46,7 +45,7 @@ export default function SessionItem({
   const validPassword = session?.factors?.password?.verifiedAt;
   const validPasskey = session?.factors?.webAuthN?.verifiedAt;
   const stillValid = session.expirationDate
-    ? session.expirationDate.toDate() > new Date()
+    ? timestampDate(session.expirationDate) > new Date()
```

Comment:
**Important:** use `timestampDate` to convert to `Date` when the value is not `undefined`